docs/get-started/README.md

```diff
@@ -18,7 +22,7 @@ You can have a maximum of **3** projects on the free plan, after you need to upg
 ## Inside the project
 ![Jexia Services](./services.png)
 1. **Collaboration** - Within this section, you can coordinate access to your project. For example, you can invite a team member or a developer to be a collaborator on the same project. This will allow them access to the project from their Jexia account. 
-1. **Settings** - Within this section, you can find the basic settings and information for the project (project name, description and URL). You may also delete the project within this section.
+1. **Settings** - Within this section, you can find the basic settings and information for the project (project name, description, and URL). You may also delete the project within this section.
```

Comment:
the , doesn't need to be added since it's the last item in the list.

Reply:
fixed